File: Python/Exercicios/URI/URI1848-Vitória.py
#URI Exercício 1848 por Vitória A Guardieiro

n = 0

while n<3:
	n = n+1
	soma = 0
	I = input()
	while I!='caw caw':
		if I[2]=='*':
			soma = soma+1
		if I[1]=='*':
			soma = soma+2
		if I[0]=='*':
			soma = soma+4
		I = input()
	print(soma)

File: C/URI/URI1848-Vitória.c
//URI Exercício 1848 em C por Vitória A Guardieiro

#include <stdio.h>
#include <string.h>

main(){
	int num, pisca;
	pisca = 0;
	char acao[10], cmp[] = "caw caw";	
	while(pisca<3){
		gets(acao);
		if(strcmp(acao,cmp)!=0){
			if(acao[0]==42){
				num = num + 4;   
			}
			if(acao[1]==42){
				num = num + 2;   
			}
			if(acao[2]==42){
				num = num + 1;   
			}
		}
		else{
			printf("%d\n",num);   
			num = 0;
			pisca++;                  
		}
	}
}
